### PR TITLE
Introduce & Refactor: Libplanet.Store

### PR DESCRIPTION
**Pull request by OnedgeLee**
As a afterwork of https://github.com/planetarium/libplanet/pull/3300, classes on former `Libplanet.Store` namespace in project `Libplanet` has been moved to new project, `Libplanet.Store`.

### CODE REVIEW

File: Libplanet.sln [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26124.0
+# Visual Studio Version 17
+VisualStudioVersion = 17.5.33424.131
```
Comment (greymistcube) [inline Libplanet.sln:4]: It probably doesn't matter much, but maybe we should keep this intact for now and have only necessary changes to the `sln` file. 🤔

File: Libplanet.Store/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,4 @@
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Libplanet.Extensions.Cocona.Tests")]
+[assembly: InternalsVisibleTo("Libplanet.Tests")]
```
Comment (greymistcube) [inline Libplanet.Store/AssemblyInfo.cs:4]: Hopefully we can get rid of this at some point. 😥

File: Libplanet.sln (hunk 1 above)
Comment (OnedgeLee) [reply Libplanet.sln]: Applied by amending 9516a107b9a91c08e749ce967d01d45d6ea66b84.

File: Libplanet.Store/AssemblyInfo.cs (hunk 2 above)
Comment (riemannulus) [reply Libplanet.Store/AssemblyInfo.cs]: https://github.com/planetarium/libplanet/pull/3303/commits/4c952a55214451fc516f76c28d8a498329ead74c Amended cc. @OnedgeLee

**Comment by riemannulus** [comment]
https://github.com/planetarium/libplanet/pull/3303/commits/87e4f8142eeb927fe2986cc4969b0322bab14c6a This is not directly coupled with this PR but I surely this commit is not important for individual PR.

**Review by greymistcube — DISMISSED** [review]

**Review by greymistcube — DISMISSED** [review]

**Review by riemannulus — APPROVED** [review]

**Pull request merged**